schema-mapping/feature-types/appearance/Appearance.json

```diff
@@ -0,0 +1,96 @@
+{
+  "identifier": "app:Appearance",
+  "table": "appearance",
+  "properties": [
+    {
+      "name": "id",
+      "namespace": "http://3dcitydb.org/3dcitydb/core/5.0",
+      "value": {
+        "column": "objectid",
+        "type": "string"
+      }
+    },
+    {
+      "name": "identifier",
+      "namespace": "http://3dcitydb.org/3dcitydb/core/5.0",
+      "value": {
+        "column": "identifier",
+        "type": "string"
+      }
+    },
+    {
+      "name": "codeSpace",
+      "namespace": "http://3dcitydb.org/3dcitydb/core/5.0",
+      "value": {
+        "column": "identifier_codespace",
+        "type": "string"
+      },
+      "parent": 1
+    },
+    {
+      "name": "theme",
+      "namespace": "http://3dcitydb.org/3dcitydb/core/5.0",
```

Comment:
should be the appearance namespace `http://3dcitydb.org/3dcitydb/appearance/5.0`

Reply:
Fixed with d3d6d348f7a3a4092390b94a26aaeb7ffee62df0